roles/candlepin/tasks/main.yml

```diff
@@ -1,4 +1,54 @@
 ---
+- name: Create log directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+    mode: '0755'
+  loop:
+  - /var/log/candlepin
+  - /var/log/tomcat
+
+- ansible.builtin.include_tasks:
+    file: certs.yml
+
+- name: Create Candlepin configuration
+  containers.podman.podman_secret:
+    state: present
+    name: candlepin-candlepin-conf
```

Comment:
Wouldn't this be candlepin(role)-candlepin(app)-candlepin(file)-conf?

(It probably shouldn't, for my sanitys sake, but that's how I read your naming convention)

I wonder if the role prefix is necessary for all the cases where rolename==appname?

Reply:
> Wouldn't this be candlepin(role)-candlepin(app)-candlepin(file)-conf?
> 
> (It probably shouldn't, for my sanitys sake, but that's how I read your naming convention)
> 
> I wonder if the role prefix is necessary for all the cases where rolename==appname?

This ^ and I updated the README to indicate so